Makefile

```diff
@@ -26,7 +26,7 @@ DEBUG_SERVICE_PORT := $(or ${DEBUG_SERVICE_PORT},40000)
 SERVICE := $(or ${SERVICE},${ASSISTED_ORG}/assisted-service:${ASSISTED_TAG})
 IMAGE_SERVICE := $(or ${IMAGE_SERVICE},${ASSISTED_ORG}/assisted-image-service:${ASSISTED_TAG})
 ASSISTED_UI := $(or ${ASSISTED_UI},${ASSISTED_ORG}/assisted-installer-ui:${ASSISTED_TAG})
-PSQL_IMAGE := $(or ${PSQL_IMAGE}, ${ASSISTED_ORG}/postgresql-12-centos7:latest)
+PSQL_IMAGE := $(or ${PSQL_IMAGE},quay.io/centos7/postgresql-12-centos7:latest)
```

Comment:
Is that something that supported by RedHat? 
I think that in our org we had a very specific version

Reply:
Tracking down the image yields https://github.com/sclorg/postgresql-container
And it seems legit: RedHat contributors, u/s d/s terminology, etc.

As for the version, you mean specific OS/psql version? Because it is pinned on psql 12 and centos 7

Reply:
as long as it's valid and supported by red-hat i'm fine.